nautobot/core/ui/object_detail.py

```diff
@@ -732,6 +752,26 @@ def render_label(self, context):
             return bettertitle(context["object"]._meta.verbose_name)
         return super().render_label(context)
 
+    def render_value(self, key, value, context):
+        try:
+            field_instance = context["obj"]._meta.get_field(key)
+        except FieldDoesNotExist:
+            field_instance = None
+
+        if key == "_hierarchy":
+            return render_tree_queryset(value)
+
+        if isinstance(field_instance, URLField):
+            return hyperlinked_field(value)
+
+        if isinstance(field_instance, BooleanField):
+            return render_boolean(value)
```

Comment:
Might make sense to move this to `KeyValueTablePanel` and just do a `if isinstance(value, bool)` instead?